packages/components/src/components/date-of-birth-picker/date-of-birth-picker.jsx

```diff
@@ -2,28 +2,19 @@ import React from 'react';
 import { toMoment } from '@deriv/shared';
 import DatePicker from '../date-picker';
 
-class DateOfBirthPicker extends React.Component {
-    state = {
-        max_date: toMoment().subtract(18, 'years'),
-        min_date: toMoment().subtract(100, 'years'),
-        value: this.props.value,
-    };
-
-    render() {
-        const { min_date, max_date, value, ...props } = this.props;
-
-        return (
-            <DatePicker
-                calendar_view='year'
-                display_format='DD-MM-YYYY'
-                max_date={this.state.max_date}
-                min_date={this.state.min_date}
-                value={this.state.value}
-                readOnly
-                {...props}
-            />
-        );
-    }
-}
+const DateOfBirthPicker = props => {
+    const [max_date] = React.useState(toMoment().subtract(18, 'years'));
+    const [min_date] = React.useState(toMoment().subtract(100, 'years'));
+    return (
+        <DatePicker
+            calendar_view='year'
+            display_format='DD-MM-YYYY'
+            max_date={max_date}
+            min_date={min_date}
+            readOnly
+            {...props}
```

Comment:
From previous code seems `max_date` and `min_date` also exist on props so it's like we're passing these values twice. Right?

Reply:
yes, but `max_date` and `min_date` props are not passed to `DateOfBirthPicker` component anywhere.